app/templates/view_supplier_users.html

```diff
@@ -23,37 +23,32 @@
 {% endblock %}
 
 {% block main_content %}
-
-
-  {% with heading = supplier.name %}
-  {% include "toolkit/page-heading.html" %}
-  {% endwith %}
-
+{% set flash_messages_content = {
+        'user_invited': 'User invited',
+        'user_not_invited': 'Not invited',
+        'user_moved': 'User moved to this supplier',
+        'user_not_moved': 'User not moved to this supplier - please check you entered the address of an existing supplier user'
+    }
+%}
 
   {% with messages = get_flashed_messages(with_categories=true) %}
   {% if messages %}
-  {% for category, message in messages %}
-  {% if category == 'error' %}
-    <div class="banner-destructive-without-action">
-  {% elif category == 'success' %}
-    <div class="banner-success-without-action">
-  {% endif %}
-        <p class="banner-message">
-  {% if message == 'user_invited' %}
-      User invited
-  {% elif message == 'user_not_invited' %}
-      Not Invited
-  {% elif message == 'user_moved' %}
-      User moved to this supplier
-  {% elif message == 'user_not_moved' %}
-      User not moved to this supplier - please check you entered the address of an existing supplier user
-  {% endif %}
-        </p>
-    </div>
-  {% endfor %}
+      {% for category, message in messages %}
+          {%
+          with
+              message = flash_messages_content[message],
+              type = "destructive" if category == 'error' else 'success'
```

Comment:
Do we use destructive-type messages when there is an error? Shouldn't there be a separate error-type message, since we are not really deleting anything?

Reply:
Yes it's an established pattern to use these banners for error messages (e.g. validation errors submitting declaration or services during applications).  This is definitely a legit use of the macro.

Reply:
Ok, thanks a lot for clarifying!